### PR TITLE
feat: GIDK-#2bbvtz | Shalinee-Prateek | Update new design of sidebar and other page

### PR DESCRIPTION
**Pull request by prateek-giddh**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
https://app.clickup.com/t/2bbvtz

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/assets/css/color.scss [hunk 1]
```diff
@@ -124,4 +125,13 @@ $linen: #fff3e8;
...
 $denim: #2c6ebd;
 $lavender-blue: #bdc3ff;
-$medium-gray: #ebeef0
\ No newline at end of file
+$medium-gray: #ebeef0;
+
+/* new design colors */
+$black-pearl: #212528;
+$trout: #494D50;
+$solitude: #eceef0;
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/assets/css/color.scss:133]: @prateek-giddh solitude color is already defined in line no. 70. Please remove if these two colors has less difference or add some other variable name for this color.

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.scss [hunk 2]
```diff
@@ -0,0 +1,558 @@
...
+                    }
+                }
+                .company-create {
+                    color: $chetwode-blue;
+                }
+                div {
+                    input {
+                        padding-left: 30px;
+                        border-left: none;
+                        // background-image: url(../../../assets/images/Search.svg);
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.scss:338]: remove commented code

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.scss [hunk 3]
```diff
@@ -0,0 +1,558 @@
...
+                    justify-content: center;
+                    align-items: center;
+                    padding: 0;
+                    margin-top: 15px;
+                    font-family: "opensans-semibold";
+                }
+             }
+        }
+        .account-dropdown-collapse {
+                            position: absolute !important;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.scss:380]: format code

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts [hunk 4]
```diff
@@ -0,0 +1,918 @@
...
+export class PrimarySidebarComponent implements OnInit, OnChanges {
+    /** Observable to store the branches of current company */
+    public currentCompanyBranches$: Observable<any>;
+    /** update IndexDb flags observable **/
+    public updateIndexDbSuccess$: Observable<boolean>;
+    /** Stores the branch list of a company */
+    public currentCompanyBranches: Array<any>;
+    /** Search query to search branch */
+    public searchBranchQuery: string;
+    /** Current organization type */
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts:59]: incorrect js doc

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts [hunk 5]
```diff
@@ -0,0 +1,918 @@
...
+    public activeAccount$: Observable<AccountResponse>;
+    /* This will show sidebar is open */
+    /** Stores the active company details */
+    public selectedCompanyDetails: CompanyResponse;
+    /** Current organization type */
+    public currentOrganizationType: OrganizationType;
+
+    /** store current selected company */
+    public selectedCompany: Observable<CompanyResponse>;
+    public activeFinancialYear: ActiveFinancialYear;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts:69]: missing js doc

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts [hunk 6]
```diff
@@ -0,0 +1,918 @@
...
+        private generalActions: GeneralActions,
+        private dbService: DbService,
+        private settingsBranchAction: SettingsBranchActions,
+        private loginAction: LoginActions,
+        private socialAuthService: AuthService,
+        private groupWithAction: GroupWithAccountsAction
+    ) {
+        // Reset old stored application date
+        this.store.dispatch(this.companyActions.ResetApplicationDate());
+        this.activeAccount$ = this.store.pipe(select(p => p.ledger.account), takeUntil(this.destroyed$));
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts:151]: don't use single character var in whole file

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts [hunk 7]
```diff
@@ -0,0 +1,918 @@
...
+        });
+    }
+
+    /**
+     * Company filtering method
+     *
+     * @param {*} ev Modal change event
+     * @memberof PrimarySidebarComponent
+     */
+    public filterCompanyList(ev) {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts:368]: add return type

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts [hunk 8]
```diff
@@ -0,0 +1,918 @@
...
+        }
+    }
+
+    /**
+     * Mouse enter call back on company
+     *
+     * @param {number} i Index of company entered
+     * @memberof PrimarySidebarComponent
+     */
+    public mouseEnteredOnCompanyName(i: number) {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts:865]: add return type

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts [hunk 9]
```diff
@@ -0,0 +1,918 @@
...
+
+    /**
+     * Returns the readable format name of menu item
+     *
+     * @private
+     * @param {*} url Url of menu item
+     * @returns
+     * @memberof PrimarySidebarComponent
+     */
+    private getReadableNameFromUrl(url) {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts:889]: add return type

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/assets/css/color.scss (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/css/color.scss:133]: Done

File: apps/web-giddh/src/app/all-items/all-item.component.scss [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+                border:none;
+                font-size: 16px;
+            }
+        }
+
+    }
+    .all-item-block{
+        .main-item{
+            border-bottom: 1px solid $solitude;
+            margin-bottom: 20PX;
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/all-items/all-item.component.scss:28]: Done

File: apps/web-giddh/src/app/shared/header/header.component.scss [hunk 11]
```diff
@@ -932,15 +1010,81 @@ ul.sub-list {
...
+                font-size: 22px !important;
+            }
         }
     }
 }
 @media (max-width: 767px) {
+    .fixed-padding{
+        padding-top: 80px;
+        #primary-new-header.primary-main-header{
+            background-color: #fff;
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/header/header.component.scss:1030]: Done

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.scss (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.scss:338]: Done

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.scss (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.scss:380]: Done

**Review by aditi-tiwari17 — APPROVED** [review]

**Review by shalinee-walkover — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**